types/Address/Address.d.ts

```diff
@@ -128,15 +129,63 @@ export declare class Address implements IAddress {
    *    If verification fails, ask them to double check their input; if they confirm that their data is correct, assume they know their address more correctly than the verification process.
    *
    * @see https://www.easypost.com/docs/api/node#create-and-verify-addresses
+   *
+   * @param {Object} params The parameters to create an {@link Address} with.
+   * @returns {Promise<Address>} The created and verified {@link Address}.
+   */
+  public create(params: Object): Promise<Address>;
+
+  /**
+   * Retrieve a list of all Addresses.
+   *
+   * The Address List is a paginated list of all {@link Address} objects associated with the given API Key.
+   * It accepts a variety of parameters which can be used to modify the scope.
+   * The has_more attribute indicates whether additional pages can be requested.
+   * The recommended way of paginating is to use either the before_id or after_id parameter to specify where the next page begins.
+   *
+   * @see https://www.easypost.com/docs/api/node#retrieve-a-list-of-addresses
+   *
+   * @param {Object} params - The parameters to use for the request.
+   */
+  static all(params?: IAddressListParameters): Promise<{ addresses: Address[]; has_more: boolean }>;
```

Comment:
I don't believe these should be "static" methods, as they're all called on an instance of the service they belong to.

Reply:
The typescript is reflecting the V6 rewrite, all the API functions have been changed to static.

Link: https://github.com/EasyPost/easypost-node/blob/master/src/services/address_service.js#L61

Reply:
Gotcha.

The mystery of why "static" doesn't seem to work the same way "static" does in other languages continues: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static